googler

```diff
@@ -334,6 +334,15 @@ except GetoptError as e:
     print("googler:", e)
     sys.exit(1)
 
+# Set up logging
```

Comment:
By the way, another benefit is we can also print errors with `logger.error` and warnings with `logger.warning`, which I didn't get to.